server/src/main/java/org/jboss/as/server/parsing/StandaloneXml_13.java

```diff
@@ -717,120 +701,6 @@ private void setServerName(final ModelNode address, final List<ModelNode> operat
         }
     }
 
-    void writeContent(final XMLExtendedStreamWriter writer, final ModelMarshallingContext context)
```

Comment:
There are still write methods in this file, however it is supposed to be a read-only parser, see all the `write*` method names.

Reply:
fixed